packages/evm/scripts/update_excubiae.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Smart Contract Synchronization from Excubiae
+#
+# This script provides a reliable way to keep our contracts up-to-date with the
+# excubiae source repository. It performs an automated sync by:
+#
+# 1. Performing a sparse checkout of the latest contracts
+# 2. Copying them to our local contracts directory
+#
+# Why This Approach?
+#
+# We chose this method because:
+#
+# * The NPM package is outdated and doesn't reflect the latest changes
+# * Since we use Hardhat instead of Foundry, we can't use Foundry's library management
```

Comment:
We could migrate to foundry if this would make life easier.
I have a slight preference for HH, purely because it's what I'm most comfortable in/familiar with, but happy to adapt.

Reply:
They released a new version with the refactor which makes me feel confident they will update the releases with a cadence we can follow instead of working off the master branch. My plan at this stage is to move to npm and follow their release schedule.